index.html

```diff
@@ -0,0 +1,457 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/main.css" />
+    <link rel="icon" href="./favicon.ico" />
+    <title>Aziz Mejri - Portfolio</title>
+  </head>
+  <body>
+    <header class="header">
+      <h1 id="logo">Aziz Mejri</h1>
+
+      <nav class="nav">
+        <ul class="nav-menu desktop">
+          <li class="nav-item">
+            <a href="#home">Portfolio</a>
+          </li>
+          <li class="nav-item">
+            <a href="#about">About</a>
+          </li>
+          <li class="nav-item">
+            <a href="#contact">Contact</a>
+          </li>
+          <li class="nav-item desktop">
+            <a href="#mail">
+              <img src="./assets/images/icons/Icon-Mail.svg" alt="mail" />
```

Comment:
- [ ] Well done @binhussen, :tophat: you did an amazing job on the design of your webpage, unfortunately the colour of your nav links  is different from the colour required. Take a look :eyes:  at the used colours in the requirements.
It should look like this
![Screenshot from 2022-11-25 20-22-49](https://user-images.githubusercontent.com/80458199/204039029-bf5eb795-3dd9-4eb7-bba6-7e824072924f.png)


Kindly update the colour, so it looks the same as in the **FIGMA** design. :vulcan_salute: